### PR TITLE
Add a method waitForAll to Signal API

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

File: scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/signal/SignalApiTest.java [hunk 1]
```diff
@@ -310,6 +310,34 @@ public void testWaitForAny() throws SignalApiException {
...
+    public void testWaitForAll() throws SignalApiException {
+        String signal_1 = "test_signal_11_1";
+        String signal_2 = "test_signal_11_2";
+        Set<String> signals = new HashSet<String>() {
+            {
+                add(signal_1);
+                add(signal_2);
+            }
+        };
+        long durationInMillis = 0;
```
Comment (amouhoub) [inline scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/signal/SignalApiTest.java:323]: Are you sure that you want to send signals at the same time with 0 latency ?

**Review by fviale — APPROVED** [review]
LGTM, but I don't understand the use case behind.

**Review by mklkun — APPROVED** [review]

File: scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/signal/SignalApiTest.java (hunk 1 above)
Comment (nebilbenmabrouk) [reply scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/signal/SignalApiTest.java:323]: yes :)

**Comment by nebilbenmabrouk** [comment]
> LGTM, but I don't understand the use case behind.

I agree, but it is a request from Denis.

**Pull request merged**